public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" style="height:100%;">
```

Comment:
I would move this to `src/index.css`. Better to keep your CSS in on place, and could be confusing later on due to the order of [CSS selector specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity):

> Inline styles added to an element (e.g., style="font-weight: bold;") always overwrite any styles in external stylesheets, and thus can be thought of as having the highest specificity.